tests/contrib/django/test_middleware.py

```diff
@@ -63,6 +64,51 @@ def test_middleware_trace_function_based_view(self):
         eq_(span.get_tag('http.url'), '/fn-view/')
         eq_(span.resource, 'tests.contrib.django.app.views.function_view')
 
+    def test_middleware_trace_error_500(self):
+        # ensures we trace exceptions generated by views
+        url = reverse('error-500')
+        response = self.client.get(url)
+        eq_(response.status_code, 500)
+
+        # check for spans
+        spans = self.tracer.writer.pop()
+        eq_(len(spans), 1)
+        span = spans[0]
+        eq_(span.get_tag('http.status_code'), '500')
+        eq_(span.get_tag('http.url'), '/error-500/')
+        eq_(span.resource, 'tests.contrib.django.app.views.error_500')
+        assert "Error 500" in span.get_tag('error.stack')
+
+    @modify_settings(
+        MIDDLEWARE={
+            'remove': 'ddtrace.contrib.django.TraceMiddleware',
+            'prepend': [
+                'ddtrace.contrib.django.TraceMiddleware',
+                'tests.contrib.django.app.middlewares.ThrowErrorMiddleware',
+            ],
+        },
+        MIDDLEWARE_CLASSES={
+            'remove': 'ddtrace.contrib.django.TraceMiddleware',
+            'prepend': [
+                'ddtrace.contrib.django.TraceMiddleware',
+                'tests.contrib.django.app.middlewares.ThrowErrorMiddleware',
+            ],
+        },
+    )
+    def test_middleware_throwing_error(self):
+        # ensures we trace the error generated by a middleware
+        url = reverse('fn-view')
+        response = self.client.get(url)
+        eq_(response.status_code, 500)
+
+        # check for spans
+        spans = self.tracer.writer.pop()
+        eq_(len(spans), 1)
+        span = spans[0]
+        eq_(span.get_tag('http.status_code'), '500')
+        eq_(span.get_tag('http.url'), '/fn-view/')
+        eq_(span.resource, 'tests.contrib.django.app.views.function_view')
```

Comment:
the stack trace (`error.stack`) is available in the span? if the implementation is not changed it shouldn't be here for what I saw before